modules/core_project_factory/variables.tf

```diff
@@ -165,7 +165,7 @@ variable "bucket_versioning" {
 
 variable "bucket_labels" {
   description = " A map of key/value label pairs to assign to the bucket (optional)"
-  type        = map
+  type        = map(any)
```

Comment:
terraform fmt seemed to add this

```
$ terraform version
Terraform v1.0.11
on linux_amd64
```

Reply:
Since this is a map of strings, can you change it to `map(string)`?

Reply:
makes sense, done!